blog/releases/announcing-knative-v0-17-release.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Announcing the Knative v0.17 release"
+linkTitle: "Version 0.17 release"
+date: 2020-08-24
+description: "The Knative v0.17 release announcement"
+type: "blog"
+image: knative-eventing.png
+---
+
+
+## Announcing Knative v0.17 Release
+
+As Knative maintains its predictable release cadence, it's time to cut another stable release across components. Be aware for the next release we are planning to make Kubernetes 1.17 the minimum version. Serving continues their journey with test and benchmark optimizations on the already v1 API version. For Eventing now ContainerSource and PingSource are available as v1beta1.
+
+Remember to check [the upgrade docs](https://knative.dev/docs/install/upgrade-installation/) for any upgrades concerns applicable to your current version before you upgrade to the new version v0.17
```

Comment:
I'd modify this section to remove any additional "wordiness".
To me, sentences like "As Knative maintains its predictable release cadence, it's time to cut another stable release across components." are waffle - they don't add any real information and make the doc harder to read.

It's not clear to me from this paragraph whether 1.17 is the minimum version for 0.17 or the _next_ release, i.e. 0.18?

Again, the "Serving continues...." bit doesn't add any info IMHO.

Another minor nit - I'd take "the" out of [the upgrade docs] and instead:
"Check the [upgrade documentation](link)..."

Instead of "the new version v0.17" I'd just say "the latest version".